app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/acquisitiondetails/AcquisitionDetailsController.scala

```diff
@@ -515,6 +527,49 @@ class AcquisitionDetailsController @Inject() (
     }
   }
 
+  def shouldUseRebase(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withFillingOutReturnAndAcquisitionDetailsAnswers(request) {
+      case (_, fillingOutReturn, answers) =>
+        withAssetTypeAndResidentialStatus(fillingOutReturn, answers) {
+          case (assetType, wasAUkResident) =>
+            Ok(
+              shouldUseRebasePage(
+                shouldUseRebaseForm,
+                routes.AcquisitionDetailsController.rebasedAcquisitionPrice(),
+                rebasingEligabilityUtil.getRebasingCutOffDate(assetType, wasAUkResident)
+              )
+            )
+        }
+    }
+  }
+
+  def shouldUseRebaseSubmit(): Action[AnyContent] = authenticatedActionWithSessionData.async { implicit request =>
+    withFillingOutReturnAndAcquisitionDetailsAnswers(request) {
+      case (_, fillingOutReturn, answers) =>
+        commonSubmitBehaviour(
+          fillingOutReturn,
+          answers
+        )(
+          shouldUseRebaseForm
+        )(page = shouldUseRebasePage(_, _, LocalDate.now()))( //remove
```

Comment:
there's `LocalDateUtils.today` you can use which will make sure you have the correct `Clock`

Reply:
what's the `remove` comment about?